pest/src/span.rs

```diff
@@ -61,6 +62,37 @@ impl<'i> Span<'i> {
         }
     }
 
+    /// Attempts to create a new span based on a sub-range.
+    ///
+    /// ```
+    /// use pest::Span;
+    /// let input = "Hello World!";
+    /// let world = Span::new(input, 6, input.len()).unwrap();
+    /// let orl = world.get(1..=3);
+    /// assert!(orl.is_some());
+    /// assert_eq!(orl.unwrap().as_str(), "orl");
+    /// ```
+    ///
+    /// # Examples
+    pub fn get(&self, range: impl RangeBounds<usize>) -> Option<Span<'i>> {
+        let start = match range.start_bound() {
+            std::ops::Bound::Included(offset) => *offset,
+            std::ops::Bound::Excluded(offset) => *offset + 1,
+            std::ops::Bound::Unbounded => 0,
+        };
+        let end = match range.end_bound() {
+            std::ops::Bound::Included(offset) => *offset + 1,
+            std::ops::Bound::Excluded(offset) => *offset,
+            std::ops::Bound::Unbounded => self.as_str().len(),
```

Comment:
`std` -> `core` if it exists in `core`

Reply:
[5c6a613](https://github.com/pest-parser/pest/pull/681/commits/5c6a613c504546c9417a06ec8144244a3e72a9c4)